src/tests/suite/daos_obj.c

```diff
@@ -4882,7 +4882,7 @@ int
 run_daos_io_test(int rank, int size, int *sub_tests, int sub_tests_size)
 {
 	int rc = 0;
-	char oclass[16] = {0};
+	char oclass[MAX_OBJ_CLASS_NAME_LEN + 1] = {0};
```

Comment:
Why is this one + 1?

Reply:
the test adds a '_' at position 0 to construct a string for test name i guess